osu.Game/Screens/SelectV2/CarouselItem.cs

```diff
@@ -29,6 +29,12 @@ public sealed class CarouselItem : IComparable<CarouselItem>
         /// </summary>
         public float DrawHeight { get; set; } = DEFAULT_HEIGHT;
 
+        /// <summary>
+        /// A number that defines the layer which this <see cref="CarouselItem"/> should be placed on depth-wise.
+        /// The higher the number, the farther the panel associated with this item is taken to the background.
```

Comment:
Probably should mention that this is relative to other panels, rather than "to the background" (because it doesn't make too much sense when they are never really approaching any background).